### ISSUE TITLE
Proceed to questionnaire

### ISSUE THREAD

**Issue by Aram4ik** [agilecube/TGS#19]
Proceed to questionnaire is possible only after 2nd video is watched.
Also there is no need to add this to menu

**Comment by agilecubeteam** [comment]
@Aram4ik: So, how user can go to "questionnaire" ? button ? if yes, where? 
At present user can go to "questionnaire" from menu.

**Comment by Aram4ik** [comment]
Described in issue #18

**Comment by agilecube** [comment]
Already covered in #18, so let's mark it closed along with #19.
